### ISSUE TITLE
assign_examiner.groovy:39-44: New users can also be...

### ISSUE THREAD

**Issue by 0pdd** [zerocracy/farm#1486]
The puzzle `1352-131ebec6` from #1352 has to be resolved:

https://github.com/zerocracy/farm/blob/95f9c336eec9c755547bd64e8eadc1bcdcbe5508/src/main/groovy/com/zerocracy/stk/pmo/resumes/assign_examiner.groovy#L39-L44

The puzzle was created by Krzysztof Krason on 24-Jul-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#1486](https://github.com/zerocracy/farm/issues/1486) is now in scope, role is `DEV`

**Comment by g4s8** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/zerocracy/farm/issues/1486#issuecomment-410455631))

@g4s8 The job [#1486](https://github.com/zerocracy/farm/issues/1486) assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2161515492); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0pdd** [comment]
The puzzle `1352-131ebec6` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@ee5f821e-71b5-4706-b3c9-61ba92e4e645, thread: pool-401-thread-1
java.lang.NumberFormatException[65] java.lang.NumberFormatException: For input string: "2018-08-04T15:46:00.53Z"
```

[0.28.15](https://github.com/zerocracy/farm): Issue: zerocracy/farm#1486, Action: `closed`

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1552](https://github.com/zerocracy/farm/issues/1552) is still not solved.

**Comment by g4s8** [comment]
@rultor release, tag=`0.29.1`

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/1486#issuecomment-410457325))

@g4s8 This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1486](https://github.com/zerocracy/farm/issues/1486) is [in scope](http://www.zerocracy.com/policy.html#14) for 9days
  * The role is `DEV`
  * The job is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8) for [30min](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @g4s8[/z](https://www.0crat.com/u/g4s8) will get 50% of their hourly rate for this job
  * Some equity will be [vested](http://www.zerocracy.com/policy.html#37) on completion
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @0pdd[/z](https://www.0crat.com/u/0pdd): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%231486%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by g4s8** [comment]
@rultor release, tag=`0.29.1`

**Comment by g4s8** [comment]
@0crat out

**Comment by g4s8** [comment]
@rultor release, tag=`0.29.2`

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/zerocracy/farm/issues/1486#issuecomment-410459629))

@g4s8 The job [#1486](https://github.com/zerocracy/farm/issues/1486) is now out of scope

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1649](https://github.com/zerocracy/farm/issues/1649) is still not solved; solved: [#1552](https://github.com/zerocracy/farm/issues/1552).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1666](https://github.com/zerocracy/farm/issues/1666) is still not solved; solved: [#1552](https://github.com/zerocracy/farm/issues/1552), [#1649](https://github.com/zerocracy/farm/issues/1649).

**Comment by 0pdd** [comment]
@0pdd all 3 puzzles are solved here: [#1552](https://github.com/zerocracy/farm/issues/1552), [#1649](https://github.com/zerocracy/farm/issues/1649), [#1666](https://github.com/zerocracy/farm/issues/1666).
